src/Utilities/universalfallback.jl

```diff
@@ -268,6 +281,52 @@ function MOI.get(uf::UniversalFallback, listattr::MOI.ListOfConstraintAttributes
     return list
 end
 
+# Objective
+MOI.supports(uf::UniversalFallback, ::MOI.ObjectiveSense) = true
```

Comment:
This is already covered by
https://github.com/JuliaOpt/MathOptInterface.jl/blob/74d74433c783a980fa118ffbda1fbe9f5f0f1d8c/src/Utilities/universalfallback.jl#L339